components/search/index.tsx

```diff
@@ -20,8 +21,8 @@ export default function Search({
 				onChange={onInputChange}
 			/>
 
-			<div>
-				<input type="button" onChange={onProfileChange} />
+			<div className={style['checkbox-container']}>
+				<input type="checkbox" onChange={onProfileChange} />
```

Comment:
Good a11y fix here. ❤️

Reply:
A `label` for screen reading :)

Reply:
Can you be more specific? Where would the label be and what would it say?

Reply:
Yes, I would add a `label` before the `input` so that the screen reader can tell the user what the checkbox is for. In the code, I have the `label` below the `input` but it's better to have it before the `input` for greater context before encountering the checkbox.

```js
<label htmlFor="hide-no-profile">
	Hide people missing a profile image
</label>
<input
	id="hide-no-profile"
	type="checkbox"
	onChange={onProfileChange}
	checked={hideNoImageChecked}
/>

```